thirdeye-ui/src/app/components/anomaly-list-v1/anomaly-list-v1.component.tsx

```diff
@@ -262,15 +292,17 @@ export const AnomalyListV1: FunctionComponent<AnomalyListV1Props> = ({
             rowKey="id"
             searchFilterValue={searchFilterValue}
             toolbarComponent={
-                <Box display="flex">
-                    <Button
-                        data-testid="button-delete"
-                        disabled={isActionButtonDisable}
-                        variant="contained"
-                        onClick={handleAnomalyDelete}
-                    >
-                        {t("label.delete")}
-                    </Button>
+                <Box display="flex" gridGap={12}>
+                    <Box display="flex">
```

Comment:
Is the inner `Box` with display flex needed?

Reply:
Without wrapping in a `Box`, the button looks like this, losing its horizontal padding

![Screenshot_150592393](https://user-images.githubusercontent.com/20799363/206116775-59f56bf8-0285-4f1e-a4ff-593c0638f434.png)

Wrapping it in a `Box` with `flex` lets it preserve its width

![Screenshot_150592394](https://user-images.githubusercontent.com/20799363/206116948-1c5e4bde-5baf-40dd-b973-7e6477c0b788.png)

Reply:
Oh I see